keps/0008-multi-cluster.yaml

```diff
@@ -33,20 +33,29 @@ Currently, KUTTL only supports a single test cluster. This KEP describes how we
 * Support applying resources across more than one test cluster.
 * Support asserting on resources across more than one test cluster.
 
-## Proposal
+## Proposals - TestStep API changes
+
+The proposal is to add a new setting within `TestStep`. We propose this field be called `runOnCluster` and of type `[]string`. This setting would allow the user to specify a list of the clusters set up by the `TestSuite` on which to run `TestSteps`.
```

Comment:
perhaps I'm not following now... I was seeing `runOnCluster` as 1 cluster `string` type.  here it is an array.  Is the proposal intending to have a single defined test to be responsible for more than 1 cluster?  if true... do you see a defined order? or run concurrently?

Reply:
>  Is the proposal intending to have a single defined test to be responsible for more than 1 cluster? if true... do you see a defined order? or run concurrently?

Yes, I am thinking that a `TestStep` will run on multiple clusters indeed. I was thinking that running concurrently was probably best, as I think we want synchronisation to be handled via the commencement/end of a `TestStep`.

What do you think? My rationale is that (at least for the tests I've been writing) we'd probably want most steps to run on all clusters, and it is the exception where we want the clusters to diverge. But if others have different requirements I'm keen to hear about them.